playbooks/common/openshift-node/config.yml

```diff
@@ -123,21 +123,14 @@
   - os_env_extras
   - os_env_extras_node
 
-- name: Set scheduleability
+- name: Set schedulability
   hosts: oo_first_master
   vars:
     openshift_nodes: "{{ hostvars
                          | oo_select_keys(groups['oo_nodes_to_config'])
                          | oo_collect('openshift.common.hostname') }}"
-    openshift_unscheduleable_nodes: "{{ hostvars | oo_select_keys(groups['oo_nodes_to_config'] | default([]))
-                                      | oo_collect('openshift.common.hostname', {'openshift_scheduleable': False}) }}"
     openshift_node_vars: "{{ hostvars | oo_select_keys(groups['oo_nodes_to_config']) }}"
   pre_tasks:
-  - set_fact:
-      openshift_scheduleable_nodes: "{{ hostvars
-                                      | oo_select_keys(groups['oo_nodes_to_config'] | default([]))
-                                      | oo_collect('openshift.common.hostname')
-                                      | difference(openshift_unscheduleable_nodes) }}"
```

Comment:
Since these variables aren't being used anymore for setting schedulability, you'll want to update the Set node OpenShift facts task in roles/openshift_node/tasks/main.yml with something similar to:
```
- name: Set node OpenShift facts
  openshift_facts:
    role: "{{ item.role }}"
    local_facts: "{{ item.local_facts }}"
  with_items:
...
  - role: node
    local_facts:
...
      kubelet_args: "{{ openshift_node_kubelet_args | default(None) }}"
      schedulable: "{{ openshift_schedulable | default(openshift_scheduleable) | default(None) }}"
```

This should cover both the old naming and the new naming for the variable as well.